use error_handling::front_of_error_handling;

fn main() {
    
    //front_of_error_handling::p_1_abort();

    //front_of_error_handling::p_2_backtrase();

    //front_of_error_handling::p_3_result();

    //front_of_error_handling::p_4_Err();

    //front_of_error_handling::p_5_errorkind();

    //front_of_error_handling::p_6_wrap();

    //front_of_error_handling::p_8_read_user_from_file();

    //front_of_error_handling::p_9_open();

    front_of_error_handling::p_11_Guess();

}